core/src/script/CGXP/plugins/Profile.js

```diff
@@ -227,6 +227,11 @@ cgxp.plugins.Profile = Ext.extend(gxp.plugins.Tool, {
     /** private: method[addActions]
      */
     addActions: function() {
+        this.i18nRasterLayers = [];
+        Ext.each(this.rasterLayers, function(layer) {
+            this.i18nRasterLayers.push(OpenLayers.i18n(layer));
+        }, this);
```

Comment:
I didn't have any clear question about the usage of IE8, then I prefer to be copatible with him.